runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -12806,6 +12836,18 @@ J9::Power::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&result
             }
          break;
 
+      case TR::java_lang_Math_fma_D:
+      case TR::java_lang_StrictMath_fma_D:
+         resultReg = inlineFPTrg1Src3(node, TR::InstOpCode::fmadd, cg);
+         return true;
+         break;
```

Comment:
Can you remove the `break` from here and the next case?  It is not needed, and it is inconsistent with the style of the other similar cases in this switch.

Reply:
Fixed in 05e6a71